<launch>
	<arg name="image_sub_topic_name" default="/kinect/qhd/image_color_rect" />
	<arg name="model_filename" default="$(find rail_object_detector)/libs/drfcn/model/rfcn_dcn_coco" />
	<arg name="debug" default="true" />
	<arg name="rate" default="1" />

    <node pkg="rail_object_detector" type="demo_pub.py" name="image_pubber" output="screen">
        <param name="rate" value="$(arg rate)" />
    </node>

	<node pkg="rail_object_detector" type="drfcn_node" name="drfcn_node">
		<param name="image_sub_topic_name" type="string" value="$(arg image_sub_topic_name)" />
		<param name="debug" type="bool" value="$(arg debug)" />
		<param name="model_filename" value="$(arg model_filename)" />
	</node>
</launch>
